litgpt/utils.py

```diff
@@ -445,15 +445,15 @@ def parse_devices(devices: Union[str, int]) -> int:
 def choose_logger(
     logger_name: Literal["csv", "tensorboard", "wandb"],
     out_dir: Path,
-    name: str,
+    name: str = "csv",
```

Comment:
I would suggest not setting a default here.

```suggestion
    name: str,
```